src/components/input/input-date-picker.tsx

```diff
@@ -0,0 +1,59 @@
+import "react-datepicker/dist/react-datepicker.css";
+import React from "react";
+import DatePicker from "react-datepicker";
+import styled from "styled-components";
+
+import { SkeletonTextLine } from "../skeletons/skeletons";
+
+type InputDatePickerProps = {
+  label: string;
+  selected?: string;
+  onChange: (date: Date) => void;
+};
+
+const InputDatePicker: React.FC<InputDatePickerProps> = ({
+  label,
+  selected,
+  onChange,
+}) => {
+  return (
+    <>
+      <Label>
+        {label}
+        {selected ? (
+          <DatePicker
+            selected={new Date(selected)}
+            onChange={onChange}
+            dateFormat={"yyyy-MM-dd"}
+          />
+        ) : (
+          <SkeletonTextLine fontSize={1.4} />
+        )}
+      </Label>
+    </>
```

Comment:
Pretty sure you don't need the `React.Fragment` here :).

Reply:
Sure